components/generate/complete/steps/CreateCompleteTicketStep3.tsx

```diff
@@ -218,7 +206,8 @@ function CreateCompleteTicketStep3() {
                     style={{ objectFit: "contain" }}
                     draggable={false}
                   />
-                  <TextField
+                  <Typography>{values.vipAssets[i].name}</Typography>
+                  {/* <TextField
```

Comment:
Are you  still using the contented part?

Reply:
fixed!